routes/v1.js

```diff
@@ -738,10 +753,13 @@ api.declare({
 
   // Validate deadline
   if (task.deadline.getTime() < new Date().getTime()) {
-    return res.status(409).json({
-      message:    "Task can't be scheduled past it's deadline",
-      error:      {deadline: task.deadline.toJSON()}
-    });
+    return res.reportError(
+      "ResourceExpired",
```

Comment:
I think this is a conflict...  409, not a 410.

But also it's a conflict because you're trying to schedule something that already resolved, by deadline expiration.
The resources in question (the task) is still around, it still exists, it's just in a state where scheduling it isn't legal anymore.